test/unit/util/test_cuda_sparse_matrix_vector_ordering.cpp

```diff
@@ -0,0 +1,230 @@
+#include <gtest/gtest.h>
+
+#include <micm/util/cuda_sparse_matrix.hpp>
+#include <micm/util/sparse_matrix_vector_ordering.hpp>
+
+#include "test_sparse_matrix_policy.hpp"
+#include "cuda_matrix_utils.cuh"
+
+TEST(CudaSparseVectorMatrix, ZeroMatrix)
+{
+  auto matrix = testZeroMatrix<micm::CudaSparseMatrix, micm::SparseMatrixVectorOrdering<2>>();
+
+  matrix.CopyToDevice();
+  auto param = matrix.AsDeviceParam();
+  micm::cuda::AddOneDriver(param);
+
+  for(auto& elem : matrix.AsVector())
+  {
+    EXPECT_EQ(elem, 1.0);
+  }
```

Comment:
I'm probably just not understanding something here, but if the `AddOneDriver()` adds one on the device side, how is this checking the host data without a call to `CopyToHost()`?

Reply:
Great catch!  This was creating an empty matrix so we weren't actually testing anything.  Updated to actually build a matrix with elements.